docs/annexes/annex-3/annex-3.01-pid-rulebook.md

```diff
@@ -480,7 +522,111 @@ If other CDDL representation types will be used in future versions of
+encoded PID attestations as SD-JWT allows for individual selective
+disclosure of objects and their properties. A hierarchical claim name
+structure is indicated by the notation `parent.child` in Table 3.
+
+Data element identifiers not listed in Table 3 are used as defined above.
+
+| Data Element Identifier | Claim Name | Note |
+|---|---|---|
+| `birth_date` | `birthdate` |  |
+| `age_over_18` | `age_equal_or_over.18` | Note: The `18` property name is a string.|
+| `age_over_NN` | `age_equal_or_over.NN` | Note: The NN property name is a string. |
+| `family_name_birth` | `birth_family_name` | |
+| `given_name_birth` | `birth_given_name` | |
+| `birth_place` | `place_of_birth.locality` | * |
+| `birth_country` | `place_of_birth.country` | |
+| `birth_state` | `place_of_birth.region` | |
+| `birth_city` | `place_of_birth.locality` | |
+| `resident_address` | `address.formatted` | |
+| `resident_country` | `address.country` | |
+| `resident_state` | `address.region` | |
+| `resident_city` | `address.locality` | |
+| `resident_postal_code` | `address.postal_code` | |
+| `resident_street` | `address.street_address` | * |
+| `resident_house_number` | `address.house_number` | * |
+| `gender` | `gender` | Note: Data type mismatch - JWT claims use `male`, `female` and custom text values.* |
+| `nationality` | `nationalities` | Note: Defined as an array, here containing only one entry.|
+| `issuance_date` | `iat` | |
+| `expiry_date` | `exp` | |
+
+*Table 2: Data Element Identifier to Claim Name Mapping*
+
+Fields marked with (*) indicate that there is not a one-to-one mapping
+between the data element and a suitable pre-defined JWT claim.
+This should be resolved in future versions of this document.
+
+EXAMPLE: The following example shows a PID in SD-JWT VC format.
+
+```json
+{
+    "vct": "https://memberstate.example/credential/pid",
+    "vct#integrity": "sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM",
+
+    "given_name": "Jean",
+    "family_name": "Dupont",
+    "birthdate": "1980-05-23",
+
+    "age_equal_or_over": {
+        "12": true,
+        "14": true,
+        "16": true,
+        "18": true,
+        "21": true,
+        "65": false
+    },
+    "age_in_years": 44,
+    "age_birth_year": 1980,
+
+    "address": {
+        "street_address": "123 Via Appia",
+        "locality": "Rome",
+        "region": "Lazio",
+        "postal_code": "00100",
+        "country": "IT"
+    },
+
+    "nationalities": ["FR"],
+
+    "gender": "male",
+
+    "place_of_birth": {
+        "locality": "Leipzig",
+        "region": "Saxony",
+        "country": "DD"
+    },
+
+    "cnf": {
+        "jwk": {
+            "kty": "EC",
+            "crv": "P-256",
+            "x": "52aDI_ur05n1f_p3jiYGUU82oKZr3m4LsAErM536crQ",
+            "y": "ckhZ-KQ5aXNL91R8Eufg1aOf8Z5pZJnIvuCzNGfdnzo"
+        }
+    },
+
+    "issuing_authority": "DE",
+    "issuing_country": "DE"
+}
+```
```

Comment:
I think it would be helpful to the reader to have an example of the JWT claim set of the SD-JWT.
Especially to demonstrate the requirement to have selectively disclosable the attributes of claims like `age_equal_or_over`, `address` etc.

For instance,

```json
{
    "vct": "https://memberstate.example/credential/pid",
    "vct#integrity": "sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM",
    
    "_sd": [
        "4WIgurnNMUzWOG3sbmmVbhcZScJ2coGwSbmcZrPtumA",
        "XQMnqxzAbDMg-H0XQvS99IrW7Mpl4D2zTBRyfmQgTKY",
        "YtWxyVoyG-gFNACWviWRXHsl1X4RtYSjB22Mspc4Zy0",
        "cACZqNooaasU06tJB4wbqd8a2BWrx2h-nh-QnrE4CGM",
        "fCJZUjADw2ZYmtWHy_AM-rOf9I_YnxUczsYhwpkbpaI",
        "gaxq0QkII8P_QmhGSSaiCPJRuoPIFiEYhp0ZCUP_HAk",
        "qgberZ_GO_M0luD--6ICnKoJoNsGniGMl5bP0mt1KS8",
        "v_ufEM5Ozoub5HdXM6QzPoJefxzn6-9Jp-KYIvp5K78",
        "wkt-dqQ0QJi3Mbw9_1v4PSl5Tyluctzy0BPbPasX4lI"
    ],

    "age_equal_or_over": {
        "_sd": [
            "51F7iAPg_JZLYQOZzmInfVb7Dm4A7ItKjf0DzLaFuRg",
            "8ehSQCxHC90lQQvwPqJmlsE3aBsTLXfSFdABhG1r4oY",
            "XziOO_oHFXgJO6HYVClDMQorsAOMU9P-AUqB4Ab8J6o",
            "kvFx5InkYZJJ72tEwsG27fi3jBGoatsZiVd3vA0b7Ak",
            "lQYi2wOoTefTc4rK-9Cq2mKW2Onwulk-e08f2r7SqDE",
            "vh3-Wnego_RenqKkZ_Alnteo8TVU_9Z-IyfNzaRNoOg"
        ]
    },

    "address": {
        "_sd": [
            "Fc909lFNobvjDRO4JCkkcRS24PtkAxzRarxPVJ4GorY",
            "I-da3bVCY1n6Koam7K2NMjyvTo8plzfvGeytvt7OgeM",
            "NzSDBxiSqMaFdIH92g281e47aBB3c_AXEOmVRucd3zw",
            "dpha7BGCPK1ZQyL9rFiNuEdvvSnkpGYSXfbskeK9T-I",
            "jqepCG5AgZE2zoDo_MjEZbzToSMnwZ90Oo5_3bMuYEE"
        ]
    },

    "place_of_birth": {
        "_sd": [
            "RZbCkkoc5yDveqfCZYxJqJbIKySbvGzfVfY_vwi2Srs",
            "WqbCLRC31iMI67-sdGkdyd5-e7Y-2SdDM-NOrZQR3O0",
            "pEtPxpWa4yYISJR9Po74E6b0Ij_qsVxH3IPv4xzrFNo"
        ]
    },
    
    "cnf": {
        "jwk": {
            "kty": "EC",
            "crv": "P-256",
            "x": "52aDI_ur05n1f_p3jiYGUU82oKZr3m4LsAErM536crQ",
            "y": "ckhZ-KQ5aXNL91R8Eufg1aOf8Z5pZJnIvuCzNGfdnzo"
        }
    }
}
```